[2023-11-07T04:52:58.170+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ps_etl.ps_schema_validation manual__2023-11-07T04:52:53.478550+00:00 [queued]>
[2023-11-07T04:52:58.178+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ps_etl.ps_schema_validation manual__2023-11-07T04:52:53.478550+00:00 [queued]>
[2023-11-07T04:52:58.179+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-07T04:52:58.191+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): ps_schema_validation> on 2023-11-07 04:52:53.478550+00:00
[2023-11-07T04:52:58.196+0000] {standard_task_runner.py:57} INFO - Started process 2276 to run task
[2023-11-07T04:52:58.200+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ps_etl', 'ps_schema_validation', 'manual__2023-11-07T04:52:53.478550+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/ps_etl.py', '--cfg-path', '/tmp/tmpmjm1rnzl']
[2023-11-07T04:52:58.204+0000] {standard_task_runner.py:85} INFO - Job 93: Subtask ps_schema_validation
[2023-11-07T04:52:58.249+0000] {task_command.py:416} INFO - Running <TaskInstance: ps_etl.ps_schema_validation manual__2023-11-07T04:52:53.478550+00:00 [running]> on host d2308b8de1b3
[2023-11-07T04:52:58.318+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ps_etl' AIRFLOW_CTX_TASK_ID='ps_schema_validation' AIRFLOW_CTX_EXECUTION_DATE='2023-11-07T04:52:53.478550+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-07T04:52:53.478550+00:00'
[2023-11-07T04:52:58.351+0000] {logging_mixin.py:151} INFO - Errors found. Details are written to error_sv_leh_2023-11-07-04-52-55.csv.
[2023-11-07T04:52:58.354+0000] {logging_mixin.py:151} INFO - Errors found. Details are written to error_sv_purchases_2023-11-07-04-52-55.csv.
[2023-11-07T04:52:58.363+0000] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2023-11-07T04:52:58.364+0000] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2023-11-07T04:52:59.529+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-07T04:52:59.540+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ps_etl, task_id=ps_schema_validation, execution_date=20231107T045253, start_date=20231107T045258, end_date=20231107T045259
[2023-11-07T04:52:59.580+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-07T04:52:59.604+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
